packit_service/worker/handlers/abstract.py

```diff
@@ -174,7 +174,7 @@ class TaskName(str, enum.Enum):
     installation = "task.run_installation_handler"
     testing_farm = "task.run_testing_farm_handler"
     testing_farm_results = "task.run_testing_farm_results_handler"
-    propose_downstream_comment = "task.run_propose_downstream_comment_handler"
+    # propose_downstream_comment = "task.run_propose_downstream_comment_handler"
```

Comment:
please put an inline comment here why it's being commented out

Reply:
I had a better idea, check it out now.